forms/officer-complaint-form/deployment/vars/dev.sh

```diff
@@ -13,3 +13,7 @@ CHAPTERS_DIR=OPO
 
 # Debug
 # RUN_BUNDLE_ANALYZER=true
+
+if [ "$CIRCLE_BRANCH" == "2309-new-forms" ]; then
```

Comment:
How will this get handled after merging to master/prod?

Reply:
There won't be any side effects unless the branch named "master" ever gets changed to "2309-new-forms." The vars for prod are in their own vars/prod.sh file. This was a quick fix I made for the sake of deployment. And honestly, I kind of like it and feel like I want to keep it.